crates/re_viewer/src/app.rs

```diff
@@ -1366,6 +1366,37 @@ fn memory_use_label_ui(ui: &mut egui::Ui, gpu_resource_stats: &WgpuResourcePoolS
             format_number(gpu_resource_stats.num_textures),
             format_number(gpu_resource_stats.num_buffers),
         ));
+    } else {
+        const CODE: &str = "use re_memory::AccountingAllocator;\n\
+                    #[global_allocator]\n\
+                    static GLOBAL: AccountingAllocator<mimalloc::MiMalloc> =\n    \
+                        AccountingAllocator::new(mimalloc::MiMalloc);";
```

Comment:
```suggestion
                    static GLOBAL: AccountingAllocator<std::alloc::System> =\n    \
                        AccountingAllocator::new(std::alloc::System);";
```

or we will also have to suggest adding a dependency on mimalloc